src/pages/new/index.js

```diff
@@ -121,6 +122,7 @@ export default () => {
           },
         ]}
       />
+      <Spacer heightDesktop="53px" heightMobile="19px" color="#000" />
```

Comment:
Shouldn't this be `color="var(--color-black)"`, according to your comment about using the variables we defined?

Also, where did these numbers come from? I'm counting 191 pixels between the bottom of the program card text to the top of the Video, and with 100px of padding within the Program Block itself, this should be 91px tall?

Reply:
As a completely unrelated comment about this particular spacer, it looks like we need to do something a bit more special to cover the whitespace under the Video Spotlight Block, behind the card on the right.

Reply:
ok

Reply:
Will create a ticket for it

Reply:
https://app.clubhouse.io/sheltertech/story/1747/fix-padding-and-background-for-the-video-spotlight-component